content/v2.0/reference/release-notes/influxdb.md

```diff
@@ -8,6 +8,35 @@ menu:
 weight: 101
 ---
 
+## v2.0.0-beta.13 [2020-06-25]
+
+## Features
+
+- Cancel submitted queries in the Data Explorer.
+- Extend templates with the source `file|url|reader`.
+- Collect stats on installed InfluxData community template usage.
+- Allow raw `github.com` host URLs for `yaml|json|jsonnet` URLs in pkger.
```

Comment:
Allow raw `github.com` host URLs for `yaml|json|jsonnet` URLs in InfluxDB templates.